test/e2e/specs/site-editor/site-editor-export.spec.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Site Editor Templates Export', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await Promise.all( [
+			requestUtils.activateTheme( 'emptytheme' ),
+			requestUtils.deleteAllTemplates( 'wp_template' ),
+			requestUtils.deleteAllTemplates( 'wp_template_part' ),
```

Comment:
Is this just a precaution in case any template parts are lingering around? Should we maybe add these to [the global `beforeAll` hook](https://github.com/WordPress/gutenberg/blob/0d00717bb88f30712129333b04e2e728c6248ad0/test/e2e/config/global-setup.ts#L36-L37), where we also remove all posts and blocks?

Reply:
Yes, to reduce archive size if there are any leftover templates. This one is test-specific.